packages/sdk/tasks/deposit.ts

```diff
@@ -108,6 +119,60 @@ task('deposit', 'Deposits WETH9 onto L2.')
       bedrock: true,
     })
 
+    const opBalanceBefore = await signer.provider.getBalance(
+      OptimismPortal.address
+    )
+
+    // Deposit ETH
+    console.log('Depositing ETH through StandardBridge')
+    const ethDeposit = await messenger.depositETH(utils.parseEther('1'))
```

Comment:
Doesn't this just wait for the transaction on L1 to confirm? Don't we have to call `messenger.waitForMessageReceipt(ethDeposit)`?

Reply:
I didn't think that would wait for the L2 conf